taxi/tests.py

```diff
@@ -1,3 +1,89 @@
 from django.test import TestCase
+from django.urls import reverse
+from django.contrib.auth import get_user_model
+from taxi.models import Manufacturer, Car
 
-# Create your tests here.
+
+class IndexViewTest(TestCase):
+    def setUp(self):
+        self.user = get_user_model().objects.create_user(
+            username="testuser", password="password123"
+        )
+        self.client.force_login(self.user)
+
+    def test_index_view_accessible(self):
+        response = self.client.get(reverse("taxi:index"))
+        self.assertEqual(response.status_code, 200)
+        self.assertTemplateUsed(response, "taxi/index.html")
+
+
+class ManufacturerListViewTest(TestCase):
+    def setUp(self):
+        self.user = get_user_model().objects.create_user(
+            username="testuser", password="password123"
+        )
+        self.client.force_login(self.user)
+
+        Manufacturer.objects.create(name="Toyota", country="Japan")
+        Manufacturer.objects.create(name="Honda", country="Japan")
+
+    def test_manufacturer_list_view_accessible(self):
+        response = self.client.get(reverse("taxi:manufacturer-list"))
+        self.assertEqual(response.status_code, 200)
+        self.assertTemplateUsed(response, "taxi/manufacturer_list.html")
+
+    def test_pagination(self):
+        response = self.client.get(reverse("taxi:manufacturer-list"))
+        self.assertIn("manufacturer_list", response.context)
+        self.assertEqual(len(response.context["manufacturer_list"]), 2)
+
+
+class CarListViewTest(TestCase):
+    def setUp(self):
+        self.user = get_user_model().objects.create_user(
+            username="testuser", password="password123"
```

Comment:
Ensure that the password is hashed by using `get_user_model().objects.create_user()` instead of directly assigning a plain text password.